test/internal.jl

```diff
@@ -78,3 +78,7 @@ end
 @test JLD2.size_size(256) === 2
 @test JLD2.size_size(65536) === 4
 @test JLD2.size_size(UInt64(4294967296)) === 8
+
+# test with one of its own types
+println()
```

Comment:
Why is this necessary?

Reply:
Um, obviously as a probe of your vigilance, duh. And congratulations - you passed.

Reply:
(removed...)

Reply:
(and apologies for the slops...)

Reply:
lol, no worries